### ISSUE TITLE
Inconsistent parameter loglevel/log_level

### ISSUE THREAD

**Issue by bastelfreak** [choria-io/puppet-choria#71]
the parameter name isn't consistent:

https://github.com/choria-io/puppet-choria/blob/755b017a7d93b61d46a097bb52af1f89d0527e58/manifests/init.pp#L9

https://github.com/choria-io/puppet-choria/blob/755b017a7d93b61d46a097bb52af1f89d0527e58/manifests/init.pp#L28

https://github.com/choria-io/puppet-choria/blob/755b017a7d93b61d46a097bb52af1f89d0527e58/templates/broker.cfg.epp#L2

I'm happy to provide a fix for this. Changing the parameter from `log_level` to `loglevel` would be a breaking change, but since this module has not yet been released as 1.0.0 it would be fine in my opinion. I suggest to adjust the parameter name to match the key in the config file. let me know what you think.

**Comment by ripienaar** [comment]
yes good idea, it's probably fine to change it and if we must we could accept both for a while but issue a warning - probably not worth it, lets just change it

I also have a inconsistency in the SRV setting but thats in the choria server code so harder to fix

Happy to accept PRs as always, thanks @bastelfreak
